### DISCUSSION TITLE
Mortgage calc

### DISCUSSION THREAD

**Discussion by MattMacD-TAFESA** [MattMacD-TAFESA/Tafe22S107#1] [General]
Have added code into mortgage calc branch. not quite working yet.
Feel free to take a look and let me know if you have any thoughts

**Comment by bradburym** [comment]
Looks good so far, I'll take a closer look tomorrow in class and hopefully we can get it working. I've finished the CurrencyConverter branch so we should be able to merge these two features into the Development branch once we've got yours working.
